app/src/main/java/com/example/parkingsystemjava/mvp/presenter/ParkingPresenter.java

```diff
@@ -0,0 +1,18 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.mvp.contract.ParkingContract;
+
+public class ParkingPresenter implements ParkingContract.PresenterContract{
+
+    private ParkingContract.ModelContract model;
+    private ParkingContract.ViewContract view;
+
+    public ParkingPresenter(ParkingContract.ModelContract model,ParkingContract.ViewContract view){
```

Comment:
you have some identations problems here.
remember to format (ctrl + alt + l ) every file you touch

Reply:
thanks! done